subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/SamplesToolingApiIntegrationTest.groovy

```diff
@@ -24,8 +24,10 @@ import org.gradle.integtests.fixtures.executer.GradleContextualExecuter
 import org.gradle.test.fixtures.file.LeaksFileHandles
 import org.gradle.util.TextUtil
 import org.junit.Rule
+import spock.lang.Ignore
 
 @LeaksFileHandles
+@Ignore('uses gradle libs-release repo')
```

Comment:
❌ Isn't that part of `https://repo.gradle.org/gradle/repo`? Do you know that @donat?

Reply:
That's the case: https://repo.gradle.org/gradle/webapp/#/artifacts/browse/simple/General/repo
- `repo` contains `libs` and `jcenter`
- `libs` contains `libs-releases`, `libs-milestones`, and `libs-snapshots`

Reply:
So I guess I can mirror `libs-releases`/`libs-milestones`/`libs-snapshot` to `repo`'s mirror directly because `repo` is a superset, right?